public/index.html

```diff
@@ -42,8 +42,7 @@ <h2 class="text-3xl text-orange-300">Links</h2>
             >—Make a change, write a pull request!
           </li>
           <li>
-            Chat is currently in a group message on ICE's Microsoft Teams or on
-            the
+            Chat is currently in private Slack channel named #the-bullpen, or on the
```

Comment:
Now that I put a comma here, I'm not sure it belongs.

Reply:
"Or on the Bullpen After Hours Discord server" is a dependent clause so [it should not have a comma](https://owl.purdue.edu/owl/general_writing/punctuation/commas/index.html)